frontend/src/metabase/query_builder/components/expressions/ExpressionWidgetHeader.module.css

```diff
@@ -0,0 +1,11 @@
+.Header {
+  color: var(--mb-color-text-medium);
+  padding: 1rem 1.5rem;
+  border-bottom: 1px solid var(--mb-color-border);
+  display: flex;
+  align-items: center;
+
+  .HeaderButton {
+    color: var(--mb-color-text-light);
+  }
```

Comment:
Text in `.HeaderButton` should be blue on hover.

Reply:
should be fixed

https://github.com/user-attachments/assets/1ef787d7-6d43-4eca-988a-42a6e778d10b